### ISSUE TITLE
Adding files to interal watch list

### ISSUE THREAD

**Issue by mantoni** [11ty/eleventy#651]
**Is your feature request related to a problem? Please describe.**
I can't find a way to add additional files to the 11ty watch list.

Background: I'm using browserify to bundle JavaScript that goes along with the page with an `app.11ty.js` file. Works great, except for watch mode, because I couldn't find a way to add files to the watch list. The same problem occurs with postcss and `*.css` files.

**Describe the solution you'd like**
The internal `addDependency` API looks like what I want. Can this be exposed? Or `eleventyConfig.watch('/app/**/*.js')` as suggested in #333.

**Describe alternatives you've considered**
I tried to `require` the browserify entry point so that eleventy crawls the dependencies, but there are too many browser specific objects being used to make it work in node.

**Comment by zachleat** [comment]
I believe this was implemented via #641. Docs at https://www.11ty.dev/docs/config/#add-your-own-watch-targets

**Comment by mantoni** [comment]
Thanks for letting me know. I tried the new feature but it does nothing in my case. I've tried all variations of `app`, `./app` and `/app`, but no rebuilds on file changes.

The `app` directory lives next to the 11ty directories. Maybe that is an issue?

Please let me know if I can provide any information or try something different. I would still ❤️ to have this feature. I have a workaround for browserify (see below), but nothing for postcss. Thank you 🙏 

__Versions:__

macOS Catalina v10.15.3
node v13.8.0
11ty v0.10.0

---

For anyone having the same issue: In the special case of browserify, I helped myself by adding this in my `app.11ty.js` file that generates the browserify bundle:

```js
  const b = browserify('./app');
  b = b.plugin('watchify');
  b.on('update', () => {
    // Initiate re-bundling by touching this file:
    const time = new Date();
    fs.utimesSync(__filename, time, time);
  });
```
